i18n(pt-BR): translate `referece/errors/prerender-client-address-not-available.mdx` (#11228)
withastro/docs: src/content/docs/pt-br/reference/errors/prerender-client-address-not-available.mdx

---

> title: Astro.clientAddress não pode ser usado dentro de rotas pré-processadas.

Reviewer: ```suggestion
title: Astro.clientAddress não pode ser usado dentro de rotas pré-renderizadas.
```

i18nReady: true
githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
---

> > **PrerenderClientAddressNotAvailable**: `Astro.clientAddress` não pode ser usado dentro de rotas pré-processadas.

Reviewer: ```suggestion
> **PrerenderClientAddressNotAvailable**: `Astro.clientAddress` não pode ser usado dentro de rotas pré-renderizadas.
```

## O que deu errado?

> A propriedade `Astro.clientAddress` não pode ser usada dentro de rotas pré-processadas.
> 
> **Veja também:**
> -  [Processamento Sob Demanda](/pt-br/guides/on-demand-rendering/)

Reviewer: ````suggestion
A propriedade `Astro.clientAddress` não pode ser usada dentro de rotas pré-renderizadas.

**Veja também:**
-  [Renderização Sob Demanda](/pt-br/guides/on-demand-rendering/)
````
